tests/test_deltalake.py

```diff
@@ -97,6 +98,63 @@ def test_added_field(self):
         self.store(self.df(b={"one": 1, "two": 2}))
         self.assert_lake_equal(self.df(a={"one": 1}, b={"one": 1, "two": 2}))
 
+    def test_last_updated_support(self):
```

Comment:
A) probably not, since this field is an `instant` field per the spec and that means it must provide at least second accuracy (though can optionally provide subsecond accuracy)

B) but why not, I threw a line in this test for it - it seems to be handled OK